### ISSUE TITLE
Players using 1.8 Forge kick players that use 1.7.x

### ISSUE THREAD

**Issue by TheRPunkt** [WaterfallMC/Travertine#69]
Hey, I'm currently setting up a server with Travertine. Until now everything worked fine, however a new builder that joined our server used 1.8 with Forge that caused all other players that use 1.7.x to get kicked out of the server. 

There are no real log errors. The kicked clients receive: 
`"[Proxy] Lost connection to server."`

The Travertine console log prints the following:
```
[11:59:58] [Netty Worker IO Thread #1/INFO]: [1.8player|/88.130.110.208:64494] <-> ServerConnector [hub] has connected
[11:59:58] [Netty Worker IO Thread #0/INFO]: [1.7player] disconnected with: [Proxy] Lost connection to server.
```
When utilizing a non-forge 1.8 version everything works fine. I also do not believe that it is a plugin related issue because i tried to delete all plugins on the used spigot server - with no effect. 
The Spigot version I am using is:
`git-Spigot-1649 (MC: 1.7.10) (Implementing API version 1.7.10-R0.1-SNAPSHOT)`

I hope someone is able to help me. If you need any additional information, feel free to ask.

**Comment by TheRPunkt** [comment]
Oh I missed that, sorry. Following error is printed to the specific spigot server:
```
[Server thread/INFO]: 1.8player[/88.130.110.208:52292] logged in with entity id 120886 at ([lobby] -0.48074271783754485, 106.0, 0.4765501389913074)
[Server thread/INFO]: 1.7player lost connection: Internal Exception: net.minecraft.util.io.netty.handler.codec.EncoderException: java.lang.RuntimeException: An internal error occured.
```
Do you think it could be a spigot issue? I wasn't sure..

**Comment by electronicboy** [comment]
Yes, that isn't a travertine issue but is likely going to be down to your allowance of multiple versions on a single server.

**Comment by TheRPunkt** [comment]
How do you mean that? Isn't Travertine supposed to support 1.7 and 1.8 clients? I tried reproducing the issue on a plain 1.7.10 spigot server without any proxy/Bungeecord. There everything worked fine, so I still suppose that it is an issue caused by Travertine or could you provide a more detailed description so that I can understand?

**Comment by electronicboy** [comment]
Travertine is a proxy that supports 1.7.10 -> 1.12.2, connections are basically exclusive pipelines where one pipeline doesn't see the other.

Spigot, however;  only supports a single version of MC, having multiple versions of clients on a single server is unsupported by the server itself, and is also unsupported by us. Any issues you have due to these types of setup are entirely on you, if you have a fix for the issue you're more than welcome to PR one, but we're not going to spend hours upon hours trying to track down issues which might not even be ours.

**Comment by TheRPunkt** [comment]
Ok, thanks for the fast and nice support even though you could not help me.

**Comment by TheRPunkt** [comment]
Just to let you all know. I tried to use an older version of BungeeCord that allowed 1.7 and 1.8 clients. Now the issue is solved, so I guess it is a issue of Travertine. I'm also guess that I'm not expierienced enough to find the issue for myself, but I will try it if I have the time.

**Comment by toxamin** [comment]
instead of using spigot 1649 try using paperspigot 1.7.10
